KBaseGenomeAnnotations/KBaseGenomeAnnotations.spec

```diff
@@ -546,14 +580,6 @@ typedef structure {
 } GenomeAnnotation; 
 
 
-
-/* 
-Reference to an GenomeAnnotation object 
-    @id ws KBaseGenomeAnnotations.GenomeAnnotation
-*/ 
-typedef string genome_annotation_ref;
```

Comment:
Already declared above. Little worried that apparently this got past the workspace type checker... it's severely out of date